src/assets/SASSAsset.js

```diff
@@ -65,6 +65,9 @@ class SASSAsset extends Asset {
 
   collectDependencies() {
     for (let dep of this.ast.stats.includedFiles) {
+      if (os.platform() === 'win32') {
```

Comment:
What about moving this to:
https://github.com/parcel-bundler/parcel/blob/master/src/Bundler.js#L394
and
https://github.com/parcel-bundler/parcel/blob/master/src/Bundler.js#L704

This way this applies to every asset type, not just scss

Reply:
maybe you could use `path.normalize` for this https://nodejs.org/api/path.html#path_path_normalize_path

Reply:
I just did another commit, moving the fix in Bundler.js like suggested and using path.normalize. I'm not sure I put it in the right positions, but as far as I could test it, it works. Thanks for your efforts!